docs/configuration.md

```diff
@@ -309,12 +309,18 @@ might contain additional Python packages as well.
 
 Create the virtualenv inside the project's root directory.
 
-If not set explicitly, `poetry` by default will create virtual environment under
-`{cache-dir}/virtualenvs` or use the `{project-dir}/.venv` directory when one is available.
+If not set explicitly, `poetry` by default will create a virtual environment under
+`{cache-dir}/virtualenvs` or use the `{project-dir}/.venv` directory if one already exists.
 
 If set to `true`, the virtualenv will be created and expected in a folder named
 `.venv` within the root directory of the project.
 
+{{% note %}}
+If a virtual environment has already been created for the project under `{cache-dir}/virtualenvs`, setting this variable to `true` will not cause `poetry` to create or use a local virtual environment.
+
+In order for this setting to take effect for a project already in that state, you must delete the virtual environment folder located in `{cache-dir}/virtualenvs`.
```

Comment:
Should we add that you can get the exact path via `poetry env info --path` (if there is such an environment)?

Reply:
That seems reasonable. Should I do a new pull request? Or is there a way to add that change into this one? 

On a related note, a couple things I came across while testing this.
1. You can't use 'poetry env remove' to get rid of the non-local virtualenv folder, at least in my (admittedly extremely limited) testing. I didn't  recommend this in the note, so no action needed here. Just mentioning it.

2. While it is possible to override the configuration of virtualenvs.in-project by setting the environment variable POETRY_VIRTUALENVS_IN_PROJECT, it's *not* possible to get the default behavior by doing so - because default behavior (of using local .venv if present, creating/using cache-dir virtualenv otherwise) only occurs when the setting is unset. It's also not possible to override global configuration to use default behavior by using a local configuration (because "poetry config virtualenvs.in-project --unset --local" just leaves the global setting in effect).
I think this is a bug, but maybe not worth addressing since nobody has run into problems because of it (?).

Reply:
> That seems reasonable. Should I do a new pull request? Or is there a way to add that change into this one?

You can just add another commit to the git branch of your PR.

Re 1.:
This might be a bug. It may depend on the setting of `virtualenv.in-project` and the existence of a local .venv. If there is no issue yet and you can provide a sequence of commands to reproduce it, you can create an issue.

Re 2.:
I think it's debatable whether this is a bug or just a missing feature. IMO, if you want to override a global specification or set a specific behavior, you can decide if you want to have in-project venvs or not, hence, set it to true or false. The default behavior is only a fallback if no decision has been made.

Reply:
With further testing I wasn't able to reproduce #1. And you're right about #2. I don't think there's any point in trying to make an obscure case work when the same task is easily achievable through existing routes.